datastore/datastore_manager.go

```diff
@@ -17,23 +17,30 @@ package datastore
 import (
 	"context"
 	"fmt"
+	"os"
+
+	"cloud.google.com/go/datastore"
 	"github.com/google/ts-bridge/storage"
-	"google.golang.org/appengine/datastore"
-	"google.golang.org/appengine/log"
+	log "github.com/sirupsen/logrus"
 )
 
 // New initializes the Manager struct implementing a generic storage.Manager interface
-func New() *Manager {
-	return &Manager{}
+func New(ctx context.Context) *Manager {
+	dsClient, err := datastore.NewClient(ctx, fetchProjectID())
+	if err != nil {
+		fmt.Errorf("could not create datastore client: %v", err)
```

Comment:
This should probably fail more catastrophically. If err is set, you don't know what dsClient will be returned.

Reply:
Agreed. Done.